content/applications/finance/payment_providers/asiapay.rst

```diff
@@ -28,13 +28,14 @@ Configuration on Odoo
 
 #. :ref:`Navigate to the payment provider AsiaPay <payment_providers/add_new>` and change its state
    to :guilabel:`Enabled`.
-#. | In the :guilabel:`Credentials` tab, fill the :guilabel:`Merchant ID`, :guilabel:`Currency`, and
-     :guilabel:`Secure Hash Secret` with the values you saved at the step
-     :ref:`payment_providers/asiapay/configure_dashboard`.
+#. | In the :guilabel:`Credentials` tab, fill in the :guilabel:`Merchant ID`,
+     :guilabel:`Secure Hash Secret`, and the :guilabel:`Currency` with the values you saved at the
```

Comment:
```suggestion
     :guilabel:`Secure Hash Secret`, and the :guilabel:`Currency` with the values you saved at the
```
Initially, it was written "[...] :guilabel:`Currency` in the `Configuration` tab [...]", but unless it is different on your DB, the "Currency" field is already in the "Credentials" tab.

Reply:
No, it is now in the `Configuration` tab. It was sometimes in `Configuration`, sometimes in `Credentials`. We feel like it makes more sense to standardize, and `Configuration` is the best place for this field, so we moved it there